Getting "Electron Error" (TypeError: Log.log is not a function)

[2021-01-31 01:12:18.387] [LOG]    Connecting socket for: MMM-Peloton
[2021-01-31 01:12:18.387] [ERROR]  App threw an error during load
[2021-01-31 01:12:18.388] [ERROR]  TypeError: Log.log is not a function
    at Class.start (/home/pi/MagicMirror/modules/MMM-Peloton/node_helper.js:7:7)
    at /home/pi/MagicMirror/js/app.js:230:18
    at new Server (/home/pi/MagicMirror/js/server.js:84:3)
    at /home/pi/MagicMirror/js/app.js:223:18
    at loadNextModule (/home/pi/MagicMirror/js/app.js:172:5)
    at /home/pi/MagicMirror/js/app.js:167:6
    at Class.loaded (/home/pi/MagicMirror/js/node_helper.js:19:3)
    at loadModule (/home/pi/MagicMirror/js/app.js:148:6)
    at loadNextModule (/home/pi/MagicMirror/js/app.js:165:5)
    at /home/pi/MagicMirror/js/app.js:167:6
[2021-01-31 01:12:18.389] [LOG]    Whoops! There was an uncaught exception...
[2021-01-31 01:12:18.390] [ERROR]  TypeError: Log.log is not a function
    at Class.start (/home/pi/MagicMirror/modules/MMM-Peloton/node_helper.js:7:7)
    at /home/pi/MagicMirror/js/app.js:230:18
    at new Server (/home/pi/MagicMirror/js/server.js:84:3)
    at /home/pi/MagicMirror/js/app.js:223:18
    at loadNextModule (/home/pi/MagicMirror/js/app.js:172:5)
    at /home/pi/MagicMirror/js/app.js:167:6
    at Class.loaded (/home/pi/MagicMirror/js/node_helper.js:19:3)
    at loadModule (/home/pi/MagicMirror/js/app.js:148:6)
    at loadNextModule (/home/pi/MagicMirror/js/app.js:165:5)
    at /home/pi/MagicMirror/js/app.js:167:6
[2021-01-31 01:19:40.289] [LOG]    MagicMirror will not quit, but it might be a good idea to check why this happened. Maybe no internet connection?
[2021-01-31 01:19:40.290] [LOG]    If you think this really is an issue, please open an issue on GitHub: https://github.com/MichMich/MagicMirror/issues
[2021-01-31 01:19:40.307] [LOG]    Whoops! There was an uncaught exception...
[2021-01-31 01:19:40.308] [ERROR]  Error: listen EADDRINUSE: address already in use 0.0.0.0:8080
    at Server.setupListenHandle [as _listen2] (net.js:1226:14)
    at listenInCluster (net.js:1274:12)
    at doListen (net.js:1413:7)
    at processTicksAndRejections (internal/process/task_queues.js:84:9) {
  code: 'EADDRINUSE',
  errno: 'EADDRINUSE',
  syscall: 'listen',
  address: '0.0.0.0',
  port: 8080

--- a/node_helper.js
+++ b/node_helper.js
@@ -4,7 +4,7 @@ const Log = require("../../js/logger");
 
 module.exports = NodeHelper.create({
 	start: function () {
-		Log.log("Starting node helper for: " + this.name);
+		this.debug("Starting node helper for: " + this.name);
 
 		this.instances = [];
 		this.peloton_api_url = "https://api.onepeloton.com/";
@@ -101,7 +101,11 @@ module.exports = NodeHelper.create({
 				prefix = "[" + this.name + "] ";
 			}
 
-			Log.log(prefix + string_to_log);
+			if (typeof Log !== "undefined" && typeof Log.log === "function") {
+				Log.log(prefix + string_to_log);
+			} else {
+				console.log(prefix + string_to_log);
+			}
 		}
 	},
 
@@ -167,7 +171,7 @@ module.exports = NodeHelper.create({
 		let self = this;
 		let instance = this.getInstance(instance_identifier);
 
-		this.debug("Logging in as: " + instance.config.username);
+		this.debug("Logging in as: " + instance.config.username, instance_identifier);
 		
 		if (instance.peloton_session_id) {
 			this.debug("Already logged in", instance_identifier);

